.github/workflows/gh-pages-rebuild.yml

```
@@ -4,7 +4,7 @@ on:
   workflow_run:
     workflows: ["Experimental Release"]
     types: [completed]
-    branches: [master]
+    branches: [master, 0.H-branch]
```

This isn't going to actually work here since it's a workflow_run, must update it on master.
```suggestion
    branches: [master]
```